network/ieee802_11_prism.ksy

```diff
@@ -0,0 +1,63 @@
+meta:
+  id: ieee802_11_prism
+  title: prism II headers for 802.11 capture
+  license: Unlicense
+  endian: le
+  encoding: ascii
+
+doc-ref:
+  - https://www.tcpdump.org/linktypes/LINKTYPE_IEEE802_11_PRISM.html
+
+seq:
+  - id: message_code
+    type: u4
+  - id: size
+    type: u4
+  - id: device_name
+    type: strz
+    size: 16
+  - id: fields
+    type: fields
+    size: fields_size
+  - id: body
+    size-eos: true
+instances:
+  header_size:
+    value: 4 + 4 + 16
+  fields_size:
+    value: size - header_size
+types:
+  fields:
+    seq:
+      - id: fields
+        type: field
+        repeat: eos
+    types:
+      field:
+        seq:
+          - id: flags
+            type: u1
+          - id: type
+            type: u1
+            enum: type
+          - id: unknown
+            type: u2
+            doc: usually zero
+          - id: is_supplied
+            type: u2
```

Comment:
The name seems misleading to me, I expect from all attributes starting with `is_` to be of type boolean, but this is some magic number. The spec says that

> The Status field is 0 if the value of that parameter is supplied and 1 if it is not supplied.

so I would either create an enum with these two states, or a value instance returning boolean. The name of the value instance with `boolean` type can start with `is_`, but in my opinion an attribute of any other type shouldn't. I'd rather go for the enum.

This enhancement is particularly important because the value logic of this field are counter-intuitive, 0 for `true` (**is** supplied) and 1 for `false` (**not** supplied), and the user of this KSY should be able to find out this just by looking in the KSY spec, without having to search in external sources.

Reply:
Fixed. Not as a enum, though.